src/main/java/racinggame/car/Cars.java

```diff
@@ -11,29 +10,31 @@
 
     private final List<Car> cars;
 
-    public Cars(int count) {
-        if (!Validator.isPositiveNumber(count)) {
-            throw new IllegalArgumentException("시도할 회수는 0보다 큰 숫자여야 합니다.");
-        }
+    public Cars(Count count) {
         this.cars = initCars(count);
     }
 
-    private List<Car> initCars(int count) {
-        List<Car> list = new ArrayList<>();
-        for (int i = 0; i < count; i++) {
-            list.add(new Car());
-        }
-
-        return list;
+    public static Cars manyOf(Count count) {
+        return new Cars(count);
     }
 
     public List<Car> getCars() {
         return Collections.unmodifiableList(cars);
     }
 
-    public void race(RandomNumber randomNumber) {
+    public void race(MovingStrategy strategy) {
         for (Car car : cars) {
-            car.race(randomNumber.createBetweenZeroToNine());
+            car.race(strategy);
         }
     }
+
+    private List<Car> initCars(Count count) {
+        int length = count.getCount();
```

Comment:
개인적인 의견이지만 `getCount()`도 좋은데 `intValue` 같은 메소드 명이 좀 더 명확할 것 같습니다 😄 
```suggestion
        int length = count.intValue();
```